docker-compose.yml

```diff
@@ -23,6 +23,8 @@ x-airflow-env:
     - GOOGLE_ANALYTICS_CONFIG_FILE_PATH=/usr/local/airflow/app-config/google-analytics/ga-data-pipeline.config.yaml
     - CIVICRM_API_KEY=SOME_API_KEY
     - CIVICRM_SITE_KEY=SOME_SITEKEY
+    - PATH_TO_TOGGL_API_TOKEN_FILE=SOME_TOGGLE_API_TOKEN_FILE_PATH
```

Comment:
minor: I am not sure whether I had added a comment somewhere else. For consistency I would suggest something like `TOGGLE_API_TOKEN_FILE_PATH` for the env variable name